comm_b.c

```diff
@@ -21,6 +21,10 @@
 #include "dump1090.h"
 #include "ais_charset.h"
 
+#ifndef M_PI
```

Comment:
Can you do this once, somewhere?

Reply:
error: use of undeclared identifier 'M_PI'

Reply:
You need to either pull this from an appropriate header (I don't know where BSD puts this) or define it once in a common header, not re-define it repeatedly.